assets/js/main.js

```diff
@@ -40,26 +40,33 @@ $(document).ready(function() {
   // Elements for taking the snapshot
   var canvas = document.getElementById('canvas');
   var context = canvas.getContext('2d');
+  var endpoint = 'http://localhost:4000/api';
   var imgCount;
 
   // Trigger photo take
   document.getElementById("snap").addEventListener("click", function() {
-    // context.drawImage(video, 0, 0, 320, 240);
-
-    // chrome.identity.getProfileUserInfo(function(userInfo) {
-      // $.ajax({
-      //   method: 'POST',
-      //   url: endpoint + '/enroll',
-      //   data: {
-      //     email: userInfo.email,
```

Comment:
Nop. Tengo que look into it. Todavia estoy en la calle, cuando llegue le voy a meter full